.github/workflows/build.yml

```diff
@@ -12,9 +12,14 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/cyfitech/cris-build-ubuntu2004:20220808
+      image: ghcr.io/cyfitech/cris-build-ubuntu2204:20220808
```

Comment:
改错了吧？

Reply:
是说你想用新linter？但这样会不会导致用2004的人本地不好搞啊

Reply:
因为NOLINTBEGIN/END是clang-tidy 14的新feature。

其实clang-tidy是可以自带一起跑format的，所以我觉得这里我们是不是可以就只跑一次tidy？之前因为format 14有一个格式bug我们才没升级，现在是不是可以直接用15跑他俩？只是lint的话装llvm的apt源应该没关系？

Reply:
Discussed offline, 15 doesn't make sense as it's not tested by any major distro yet. 14 is also too aggressive because it'll require people to install non-distro toolchains in most cases. We should still focus on getting the most on a widely available version and don't dig into the diminishing return of edgy versions.